core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -183,8 +184,8 @@ abstract class H2OContext private(val sparkSession: SparkSession, private val co
 
     // Initial update
     val swHeartBeatEvent = getSparklingWaterHeartBeatEvent()
-    sparkSession.sparkContext.listenerBus.post(swHeartBeatEvent)
-    sparkSession.sparkContext.listenerBus.post(H2OContextStartedEvent(h2oClusterInfo, h2oBuildInfo, swPropertiesInfo))
+    SparkSessionUtils.active.sparkContext.listenerBus.post(swHeartBeatEvent)
```

Comment:
nit: Can we extract `SparkSessionUtils.active.sparkContext.listenerBus` to a separate line?

Reply:
👍